osu.Desktop.VisualTests/Tests/TestCasePlaySongSelect.cs

```diff
@@ -35,9 +43,19 @@ public override void Reset()
 
                 db.Import(sets);
             }
-            Add(new PlaySongSelect());
+
+            Add(songSelect = new PlaySongSelect());
+            OnArtist = () => { songSelect.Filter.Sort = FilterControl.SortMode.Artist; };

```

Comment:
why even make these when you could put the anonymous delegates in the AddButton lines below?